sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -243,28 +238,28 @@ export class ManagementClient extends LinkEntity {
           sropt,
```

Comment:
Previously, we had logs for negotiating claims appear first and only on succeeding to do that we would see the logs for creating the links. With the change in this PR, that order would be reversed. If you can ensure that you log the above information inside `initLink()`, then I would recommend removing the above logging statement

Reply:
I did two things to fix this:

1. Removed the message you pointed out (we have a log message directly in init() that will dump the options to the link and it's the entire object that's passed in options which'll have the same info).
2. I also added in a "we've finished creating the link" message into init() so I was able to get rid of another post-message in the mangement link as well.